packages/scripts/assets/verifiedAnswers/mongodb-verified-answers.yaml

```diff
@@ -0,0 +1,16 @@
+- questions:
+    - How do I verify answers?
+  answer: To verify an answer...
+  author_email: someone@example.com
+  references:
+    - uri: http://example.com
+      title: Example
+  hidden: true
```

Comment:
Are we concerned about verified answers becoming outdated? Should we also have some sort of field that marks the answers as valid/verified until some condition is met? e.g.

- `valid_until: new Date("June 30 2024")`
- `valid_for_versions: ["6.2", "7.0"]`

I think the date based approach would be easiest to implement.

With that in place we could set up a cron that emails the `author_email` (and maybe us too) asking them to review their answers & either delete them or "revalidate" for another period of time.

Reply:
Captured: https://jira.mongodb.org/browse/EAI-224